cssom-view-1/Overview.bs

```diff
@@ -1161,6 +1161,9 @@ dictionary ScrollIntoViewOptions : ScrollOptions {
 dictionary CheckVisibilityOptions {
     boolean checkOpacity = false;
```

Comment:
Please add a `Note` that says that `checkVisibilityCSS` and `checkOpacity` are deprecated.

Reply:
We haven't resolved to deprecate these. I've added a note that says they are historical names that have aliases that match the new naming scheme. I suspect we want to revisit at CSSWG if we want to deprecate the names though.

Reply:
I think it's fine to call them deprecated IMO. I don't feel strongly either way tho. Assuming @smfr is fine with the current wording feel free to merge.